__tests__/helper-functions.js

```diff
@@ -18,32 +18,32 @@ const adminLogin = async user => {
     variables: {
       user: {
         email: user ? user.email : process.env.SUPER_ADMIN_EMAIL,
-        password: user ? user.password : process.env.SUPER_ADMIN_PASSWORD
+        password: user ? user.password : process.env.SUPER_ADMIN_PASSWORD,
       },
     },
   });
   return result.data.loginAdmin.token;
-  
 };
 
-const setupApp = async () => {
+const setupApp = async context => {
```

Comment:
Please, rename it to `user` because it holds the user object provided by someone who uses this method, but not the `context`

Reply:
solved